app/api/v2/handlers/objective_api.py

```diff
@@ -21,7 +21,10 @@ def add_routes(self, app: web.Application):
         router.add_patch('/objectives/{id}', self.update_objective)
         router.add_put('/objectives/{id}', self.create_or_update_objective)
 
-    @aiohttp_apispec.docs(tags=['objectives'])
+    @aiohttp_apispec.docs(tags=['objectives'],
+                          summary='Retrieve objectives',
+                          description='Retrieve all objectives by criteria. Use fields from the `ObjectiveSchema` in '
+                                      'the request body to filter retrieved objectives.')
     @aiohttp_apispec.querystring_schema(BaseGetAllQuerySchema)
     @aiohttp_apispec.response_schema(ObjectiveSchema(many=True, partial=True))
```

Comment:
I think it might be nice to add a `description` to the `aiohttp_apispec.response_schema()` for this group of endpoints. `GET /objective/{id}` and `POST /objective` will both return a single dumped Objective object, but it may be helpful for users if we specify that `GET /objectives` will be a list of Objective objects.